lib/offsite_payments/integrations/pay_vector.rb

```diff
@@ -0,0 +1,776 @@
+#!/bin/env ruby
+# encoding: utf-8
```

Comment:
Are there any UTF-8 characters in here? I took a quick look and didn't see any, and I would prefer to avoid hard coding this